_data/people/msneubauer.yml

```
@@ -150,3 +150,14 @@ presentations:
   - doma
   - as
   - ssc
+- title: Virtual Meetings Workshop Summary
+  date: 2021-08-19
+  url: https://indico.cern.ch/event/1034113/contributions/4452694/attachments/2295587/3904122/IUPAP-C11_IRIS-HEP_Virtual_Meetings.pptx.pdf
+  meeting: International Union of Pure & Applied Physics: Particles and Fields Meeting
```

```suggestion
  meeting: "International Union of Pure & Applied Physics: Particles and Fields Meeting"
```

In YAML, colon followed by a space are not allowed because it looks like a mapping, it must be quoted. (a colon without a space, such as a url, is okay).